lab_1/main.py

```diff
@@ -25,7 +33,18 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
+    try:
```

Comment:
`try ... except` blocks are usually wrap the line where the Exception is likely to be raised.

Take a look at `isinstance` for type-checking.